content/en/docs/ambient/architecture/index.md

```diff
@@ -9,15 +9,63 @@ owner: istio/wg-networking-maintainers
+| `istio.io/waypoint-for` | Alpha | `Gateway` | Specifies the waypoint's capture scope, valid value: `service` or `none` or `workload` or `all`. The default value is `service`. |
+
+### Policy attachment to waypoints
+
+You can attach Layer 7 policies (such as `AuthorizationPolicy`, `RequestAuthentication`, `Telemetry`, `WasmPlugin`, etc) to your waypoint using `targetRef`.
+
+1. To attach the entire waypoint, set `Gateway` as the `targetRef` value. The example below shows how to attach the `viewer` policy
+to the waypoint named `waypoint` for the `default` namespace:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: viewer
+  namespace: default
+spec:
+  targetRef:
+    kind: Gateway
+    group: gateway.networking.k8s.io
+    name: waypoint
+{{< /text >}}
+
+1. To attach a given service within the waypoint, set `Service` as the `targetRef` value. The example below shows how to attach
+the `productpage-viewer` policy to the `productpage` service in the `default` namespace:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: productpage-viewer
+  namespace: default
+spec:
+  targetRef:
+    kind: Service
+    group: ""
+    name: productpage
+{{< /text >}}
+
 ## Traffic routing
 
 In {{< gloss "ambient" >}}ambient mode{{< /gloss >}}, workloads can fall into 3 categories:
 1. **Uncaptured:** this is a standard pod without any mesh features enabled.
-1. **Captured:** this is a pod that has traffic intercepted by {{< gloss >}}ztunnel{{< /gloss >}}. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
+1. **Captured:** this is a pod that has traffic intercepted by {{< gloss >}}ztunnel{{< /gloss >}}. A pod's captured mode can be enabled by setting the `istio.io/dataplane-mode=ambient` label on its namespace, which enables all pods' captured mode for that namespace.
 1. **Waypoint enabled:** this is a pod that is "Captured" *and* has a {{< gloss "waypoint" >}}waypoint proxy{{< /gloss >}} deployed.
-  A waypoint will, by default, apply to all pods in the same namespace.
-  It can optionally be set to apply to only a specific service account with the `istio.io/for-service-account` annotation on the `Gateway`.
-  If there is both a namespace waypoint and service account waypoint, the service account waypoint takes precedence.
+  If a namespace is labeled with `istio.io/use-waypoint` with its default waypoint for the namespace, the waypoint will apply to all pods in the namespace.
+  The `istio.io/use-waypoint` label can optionally be set to apply to only a specific service or pod with its desired waypoint.
+  If the `istio.io/use-waypoint` label exists on the namespace and/or service and/or pod, the pod waypoint takes precedence over the service waypoint, which takes precedence over the namespace waypoint.
```

Comment:
Is this correct? @ilrudie or @howardjohn

Reply:
Technically, wl and svc attachment are equal weight because they don't actually compete with each other. there is no need to know which overwrites the other. How the traffic is addressed determines which waypoint would be used. service addressed traffic will not use a wl-attached waypoint and likewise wl addressed traffic will not use a svc-attached waypoint.

Reply:
The prob is what if my namespace says `use-waypoint: A`, and my service has `use-waypoint: B` and my workload has `use-waypoint: C` or if my service has `use-waypoint: #none` and my workload has `use-waypoint: C`?

Reply:
Cool, marking as resolved